### ISSUE TITLE
Receiving "You need to build the cli package first" when attempting to start GraphWeaver from quick start

### ISSUE THREAD

**Issue by intmainvoid** [exogee-technology/graphweaver#84]
**Describe the bug**
After following the [quick start guide](https://docs.graphweaver.com/quick-start), I receive an error when attempting to start the server.

**To Reproduce**
1. Open a terminal window on my Mac
2. Review the quick start documentation at https://docs.graphweaver.com/quick-start
3. Follow the quick start guide step by step.
    - `cd sandpit-projects`
    - `npx @exogee/graphweaver-cli create`
        - What would your like to call your new project? `gavins-graphweaver-project-no-1`
        - Which GraphWeaver backends will you need? `MikroORM - PostgreSQL Backend`
        - OK, we're ready- I'm going to create a new app in "<folder>" - is that OK? `y`
    - `cd gavins-graphweaver-project-no-1`
    - `pnpm i`
    - `pnpm start`

**Expected behavior**
The GraphWeaver development server is started and I can begin building my GraphQL API right away

**Actual behavior**
```
➜  gavins-graphweaver-project-no-1 pnpm start

> gavins-graphweaver-project-no-1@0.1.0 start /Users/gavinconway/Development/sandpit/gavins-graphweaver-project-no-1
> graphweaver start

You need to build the cli package first
```

**Screenshots or Screencasts**
N/A

**Desktop (please complete the following information):**
 - OS and version: MacOS 13.3.1a

**Additional context**
- npx version: 8.5.3
- node version: v14.19.0
- pnpm version: 6.32.12

When looking in the path: `<project folder>/node_modules/@exogee/graphweaver-cli/bin`, the `index.js` file is missing.

Full output:
```
➜  sandpit npx @exogee/graphweaver-cli create
Need to install the following packages:
  @exogee/graphweaver-cli
Ok to proceed? (y) y
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm WARN deprecated superagent@7.1.6: Please downgrade to v7.1.5 if you need IE/ActiveXObject support OR upgrade to v8.0.0 as we no longer support IE and published an incorrect patch version (see https://github.com/visionmedia/superagent/issues/1731)
npm WARN deprecated querystring@0.2.1: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
GraphWeaver

? What would your like to call your new project? gavins-graphweaver-project-no-1
? Which GraphWeaver backends will you need? 
? OK, we're ready- I'm going to create a new app in "/Users/gavinconway/Development/sandpit/gavins-graphweaver-project-no-1" - is that OK? Yes
All Done!
Make sure you to pnpm install, then pnpm start.
npm notice 
npm notice New major version of npm available! 8.5.3 -> 9.6.7
npm notice Changelog: https://github.com/npm/cli/releases/tag/v9.6.7
npm notice Run npm install -g npm@9.6.7 to update!
npm notice 
```

**Comment by kyelewis** [comment]
@intmainvoid to check if this is the only issue, can you find these three in the package.json:

```        
  "@exogee/graphweaver": "0.1.13",
  "@exogee/graphweaver-apollo": "0.1.13",
  "@exogee/graphweaver-cli": "0.1.13",
```

and update them to 0.1.14

```
  "@exogee/graphweaver": "0.1.14",
  "@exogee/graphweaver-apollo": "0.1.14",
  "@exogee/graphweaver-cli": "0.1.14",
```

then do an `pnpm i` and try running `graphweaver start` again?

**Comment by skeep-exogee** [comment]
@intmainvoid I just tested with the same node version and pnpm as you and got the same result, thanks for raising the issue!

I checked what version I was running when doing my testing and I was running:

Node `v16.19.0`
PNPM `v7.26.3`

Using these versions I was able to start the server as expected.

Could you try this for me:

- Start by installing the above dependencies versions for node and pnpm
- then remove the current project you have `rm -rf gavins-graphweaver-project-no-1`
- finally run through the install again

If this works then I will make sure to upgrade the documentation so that we have the prerequisites defined 👍

**Comment by skeep-exogee** [comment]
Hi @intmainvoid thanks for reporting.

This should now be fixed in release `0.1.16`.

Can you try again by running `npx @exogee/graphweaver-cli@latest create` and let us know if you still have an issue?

**Comment by intmainvoid** [comment]
Thanks @skeep-exogee and @kyelewis - Looks like running the `npx` command again with 0.1.16 fixed the issues. It now starts without issue.
